controllers/slice/namespaces.go

```diff
@@ -49,10 +50,16 @@ var (
 )
 
 func (r *SliceReconciler) ReconcileSliceNamespaces(ctx context.Context, slice *kubeslicev1beta1.Slice) (ctrl.Result, error, bool) {
+	log := logger.FromContext(ctx)
+	debuglog := log.V(1)
 	res, err, reconcile := r.reconcileAppNamespaces(ctx, slice)
 	if reconcile {
 		return res, err, true
 	}
+	if slice.Status.SliceConfig.SliceOverlayNetworkDeploymentMode == v1alpha1.NONET {
```

Comment:
Plz remove this check for now. Lets keep allowed ns and network policy flow as is since it applies to the cni, not nsm.